### PR TITLE
briefly update the grafana tikv dashboard doc

### LINKED ISSUE #10319: Docs of TiKV grafana dashboard is fully outdated
## Change Request

Please answer the following questions before submitting your issue. Thanks!

1. Describe what you find is inappropriate or missing in the existing docs.

This page https://docs.pingcap.com/zh/tidb/stable/grafana-tikv-dashboard#raft-process describes the metrics dashboard of the TiKV server. However, in v6.1.0, the count of metrics has increased a lot and its layout is fully refactored. Some panels are moved from other rows, and more metrics are added. 

2. Describe your suggestion or addition.
We need to rewrite this doc with more easy achievable tracing issues. Some principles should be followed while we are proceeding with this refactor.

	1. Focus on the metrics that really matter. There are too many metrics in TiKV, we need to filter out the less frequently used metrics out of the docs. Some critical cluster-level information metrics should be documented. Then some metrics that can help users or supporters to Investigation issues should be documented like the metrics in panel `tikv-trouble-shooting`.
	2.  Docs should be self-explained, instead of the translation of description in Grafana.

### PR DESCRIPTION
**Pull request by tonyxuqqi**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Update the TiKV dashboard metrics on a few important panels, to fix #10319

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v7.0 (TiDB 7.0 versions)
- [x] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: grafana-tikv-dashboard.md [hunk 1]
```diff
@@ -76,14 +76,21 @@ aliases: ['/docs-cn/dev/grafana-tikv-dashboard/','/docs-cn/dev/reference/key-mon
 
 - Raft store CPU：raftstore 线程的 CPU 使用率，通常应低于 80% * `raftstore.store-pool-size`
 - Async apply CPU：async apply 线程的 CPU 使用率，通常应低于 90% * `raftstore.apply-pool-size`
-- Scheduler worker CPU：scheduler worker 线程的 CPU 使用率，通常应低于 90% * `storage.scheduler-worker-pool-size`
+- Store writer CPU: async io线程的CPU使用率，通常应低于 90% * `raftstore.store-io-pool-size`
```
Comment (ran-huang) [inline grafana-tikv-dashboard.md:79]:
```suggestion
- Store writer CPU：async io 线程的 CPU 使用率，通常应低于 90% * `raftstore.store-io-pool-size`
```

File: grafana-tikv-dashboard.md [hunk 2]
```diff
@@ -76,14 +76,21 @@ aliases: ['/docs-cn/dev/grafana-tikv-dashboard/','/docs-cn/dev/reference/key-mon
...
 - RocksDB CPU：RocksDB 线程的 CPU 使用率
+- Coprocessor CPU：coprocessor 线程的 CPU 使用率
 - GC worker CPU：GC worker 线程的 CPU 使用率
 - BackGround worker CPU：background worker 线程的 CPU 使用率
+- Import CPU: Import线程的CPU使用率
+- Backup Worker CPU: Backup线程的CPU使用率
+- CDC Worker CPU: CDC Worker线程的CPU使用率
+- CDC endpoint CPU: Cdc endpoint CPU使用率
+- Raftlog fetch worker CPU: Async raft log fetcher worker的CPU使用率
+- TSO Worker CPU: TSO Worker线程的CPU使用率
```
Comment (ran-huang) [inline grafana-tikv-dashboard.md:93]:
```suggestion
- Import CPU：Import 线程的 CPU 使用率
- Backup Worker CPU：Backup 线程的 CPU 使用率
- CDC Worker CPU：CDC Worker 线程的 CPU 使用率
- CDC endpoint CPU：CDC endpoint 的 CPU 使用率
- Raftlog fetch worker CPU：Async raft log fetcher worker 的 CPU 使用率
- TSO Worker CPU: TSO Worker 线程的 CPU 使用率
```

File: grafana-tikv-dashboard.md [hunk 3]
```diff
@@ -113,9 +120,9 @@ aliases: ['/docs-cn/dev/grafana-tikv-dashboard/','/docs-cn/dev/reference/key-mon
     - append: ready 内 Raft log entry 的 ops
     - commit: ready 内 committed Raft log entry 的 ops
     - snapshot: 携带 snapshot 的 ready 的 ops
-- 0.99 Duration of Raft store events：99% 的 raftstore 事件所花费的时间
-- Process ready duration：处理 ready 所花费的时间
-- Process ready duration per server：每个 TiKV 实例处理 ready 所花费的时间，99.99% 的情况下，应该小于 2s
+- Max Duration of Raft store events：100% 的 raftstore 事件所花费的时间
+- Replica read lock checking duration：处理 Replica Read时检查lock所花费的时间
+- Peer msg length distribution：每个TiKV中每个region一次性处理Peer消息的个数，消息越多说明peer越繁忙。
```
Comment (ran-huang) [inline grafana-tikv-dashboard.md:125]:
```suggestion
- Replica read lock checking duration：处理 Replica Read 时检查 lock 所花费的时间
- Peer msg length distribution：每个 TiKV 中每个 region 一次性处理 Peer 消息的个数，消息越多说明 peer 越繁忙。
```

File: grafana-tikv-dashboard.md [hunk 4]
```diff
@@ -331,6 +338,30 @@ aliases: ['/docs-cn/dev/grafana-tikv-dashboard/','/docs-cn/dev/reference/key-mon
 - Ingest SST duration seconds：ingest SST 所花费的时间
 - Stall conditions changed of each CF：每个 CF stall 的原因
 
+### Raft Engine
+
+- Operations
+    - write: Raft Engine每秒写操作的次数
+    - read-entry: Raft Engine每秒读raft数据的次数
+    - read-message: Raft Engine每秒读raft消息的次数
```
Comment (ran-huang) [inline grafana-tikv-dashboard.md:346]:
```suggestion
    - write：Raft Engine 每秒写操作的次数
    - read-entry：Raft Engine 每秒读 raft 数据的次数
    - read-message：Raft Engine 每秒读 raft 消息的次数
```

File: grafana-tikv-dashboard.md [hunk 5]
```diff
@@ -331,6 +338,30 @@ aliases: ['/docs-cn/dev/grafana-tikv-dashboard/','/docs-cn/dev/reference/key-mon
...
+- Write duration：Raft Engine写操作的耗时，该耗时基本接近写入这些数据所包含的磁盘IO的latency之和
```
Comment (ran-huang) [inline grafana-tikv-dashboard.md:347]:
```suggestion
- Write duration：Raft Engine 写操作的耗时，该耗时基本接近写入这些数据所包含的磁盘 IO 的 latency 之和
```

File: grafana-tikv-dashboard.md [hunk 6]
```diff
@@ -331,6 +338,30 @@ aliases: ['/docs-cn/dev/grafana-tikv-dashboard/','/docs-cn/dev/reference/key-mon
...
+- Flow
+    - write: Raft Engine写流量
+    - rewrite append: 重写append日志的流量
+    - rewrite rewrite: 重写rewrite日志的流量
```
Comment (ran-huang) [inline grafana-tikv-dashboard.md:351]:
```suggestion
    - write：Raft Engine 写流量
    - rewrite append：重写 append 日志的流量
    - rewrite rewrite：重写 rewrite 日志的流量
```

File: grafana-tikv-dashboard.md [hunk 7]
```diff
@@ -331,6 +338,30 @@ aliases: ['/docs-cn/dev/grafana-tikv-dashboard/','/docs-cn/dev/reference/key-mon
...
+- Write Duration Breakdown(99%)
+    - wal: 写Raft Engine WAL的延迟
+    - wait: 写入前等待时间
+    - apply: apply到内存的时间
```
Comment (ran-huang) [inline grafana-tikv-dashboard.md:355]:
```suggestion
- Write Duration Breakdown (99%)
    - wal：写 Raft Engine WAL 的延迟
    - wait：写入前等待时间
    - apply：apply 到内存的时间
```

File: grafana-tikv-dashboard.md [hunk 8]
```diff
@@ -331,6 +338,30 @@ aliases: ['/docs-cn/dev/grafana-tikv-dashboard/','/docs-cn/dev/reference/key-mon
...
+- Bytes/Written 每次写入对应的bytes
+- WAL Duration Breakdown (P99%): 写WAL内部各个阶段所花的时间
```
Comment (ran-huang) [inline grafana-tikv-dashboard.md:357]:
```suggestion
- Bytes/Written 每次写入对应的 bytes
- WAL Duration Breakdown (P99%)：写 WAL 内部各个阶段所花的时间
```

File: grafana-tikv-dashboard.md [hunk 9]
```diff
@@ -331,6 +338,30 @@ aliases: ['/docs-cn/dev/grafana-tikv-dashboard/','/docs-cn/dev/reference/key-mon
...
+- File Count
+    - append: Raft Engine append数据用的文件个数
+    - rewrite: Raft Engine rewrite文件用的个数（rewrite类似于RocksDB的compaction)
```
Comment (ran-huang) [inline grafana-tikv-dashboard.md:360]:
```suggestion
    - append：Raft Engine append 数据用的文件个数
    - rewrite：Raft Engine rewrite 文件用的个数（rewrite 类似于 RocksDB 的 compaction）
```

File: grafana-tikv-dashboard.md [hunk 10]
```diff
@@ -331,6 +338,30 @@ aliases: ['/docs-cn/dev/grafana-tikv-dashboard/','/docs-cn/dev/reference/key-mon
...
+- Entry Count
+    - rewrite:  Raft Engine中rewrite过的记录条数
+    - append: Raft Engine中append的记录条数
```
Comment (ran-huang) [inline grafana-tikv-dashboard.md:363]:
```suggestion
    - rewrite：Raft Engine 中 rewrite 过的记录条数
    - append：Raft Engine 中 append 的记录条数
```

**Comment by qiancai** [comment]
Removed the needs-cherry-pick-release-6.6 label because the v6.6 docs have been archived at https://docs-archive.pingcap.com/zh/tidb/v6.6 and will no longer receive new updates.

File: grafana-tikv-dashboard.md [hunk 11]
```diff
@@ -331,6 +338,30 @@ aliases: ['/docs-cn/dev/grafana-tikv-dashboard/','/docs-cn/dev/reference/key-mon
 - Ingest SST duration seconds：ingest SST 所花费的时间
 - Stall conditions changed of each CF：每个 CF stall 的原因
 
+### Raft Engine
+
+- Operations
+    - write：Raft Engine 每秒写操作的次数
+    - read-entry：Raft Engine 每秒读 raft 数据的次数
+    - read-message：Raft Engine 每秒读 raft 消息的次数
```
Comment (overvenus) [inline grafana-tikv-dashboard.md:346]:
```suggestion
    - read_entry：Raft Engine 每秒读 raft 日志的次数
    - read_message：Raft Engine 每秒读 raft 元数据的次数
```

**Review by overvenus — COMMENTED** [review]
Rest LGTM

File: grafana-tikv-dashboard.md [hunk 12]
```diff
@@ -331,6 +338,30 @@ aliases: ['/docs-cn/dev/grafana-tikv-dashboard/','/docs-cn/dev/reference/key-mon
...
+    - wait：写入前等待时间
+    - apply：apply 到内存的时间
+- Bytes/Written 每次写入对应的 bytes
+- WAL Duration Breakdown (P99%)：写 WAL 内部各个阶段所花的时间
+- File Count
+    - append：Raft Engine append 数据用的文件个数
+    - rewrite：Raft Engine rewrite 文件用的个数（rewrite 类似于 RocksDB 的 compaction）
+- Entry Count
+    - rewrite：Raft Engine 中 rewrite 过的记录条数
+    - append：Raft Engine 中 append 的记录条数
```
Comment (ran-huang) [inline grafana-tikv-dashboard.md:363]:
```suggestion
- File Count
    - append：Raft Engine 用于 append 数据的文件个数
    - rewrite：Raft Engine 用于 rewrite 的文件个数（rewrite 类似于 RocksDB 的 compaction）
- Entry Count
    - rewrite：Raft Engine 中已经 rewrite 的记录条数
    - append：Raft Engine 中已经 append 的记录条数
```

**Pull request merged**